direnvrc

```diff
@@ -336,9 +336,10 @@ use_nix() {
     esac
   done
 
-  # nixfile may be empty,
-  # but nix_direnv_watch_file checks for existence before adding to watches
-  nix_direnv_watch_file "$HOME/.direnvrc" "$HOME/.config/direnv/direnvrc" .envrc "$nixfile"
+  nix_direnv_watch_file "$HOME/.direnvrc" "$HOME/.config/direnv/direnvrc" ".envrc"
+  if [ -e "$nixfile" ]; then
```

Comment:
Can this be something that is not a file?

```suggestion
  if [ -f "$nixfile" ]; then
```

Reply:
Maybe symlink?